src/index.ts

```diff
@@ -96,11 +96,18 @@ function fontawesomeSubset(subset: SubsetOption, outputDir: string, options: Fon
 
         const svgFile = readFileSync(svgFilePath).toString();
         const glyphsToRemove = findGlyphsToRemove(svgFile, fontFamily, icons);
-        const svgContentsNew = svgFile.replace(new RegExp(`(<glyph glyph-name="(${glyphsToRemove.join("|")})".*?\\/>)`, "gms"), "").replace(/>\s+</gms, "><");
+        const svgContentsNew = svgFile
+            .replace(/<metadata>.*<\/metadata>/s, "")
```

Comment:
Looks like this may not actually be needed? The metadata here is the build time of the _original_ SVG font - so it _should_ only ever change if you update your dependencies (fontawesome specifically). I think the `ts` option below is really the change that resolves this issue.

Reply:
I'm not sure where the dates in the metadata are coming from. But I'm not sure it makes sense to include the metadata when generating the other icon sets. The other native icon sets do not include the metadata.

I will take a look at how the native icon sets are generated and report back.